src/app/src/components/FacilitiesMap.jsx

```diff
@@ -1,519 +1,276 @@
-                            bootstrapURLKeys={{
-                                region: this.props.countryCode,
-                                key: GOOGLE_CLIENT_SIDE_API_KEY,
-                            }}
-                            options={this.createMapOptions}
-                        />
-                    </ShowOnly>
-                </div>
+    return (
+        <ReactLeafletMap
+            id="oar-leaflet-map"
+            ref={mapRef}
+            center={initialCenter}
+            zoom={isDetailsMap ? detailsZoomLevel : initialZoom}
+            renderer={L.canvas()}
+            style={mapComponentStyles.mapContainerStyles}
+            zoomControl={false}
+            maxBounds={[[-90, -180], [90, 180]]}
+            worldCopyJump
+        >
+            <ReactLeafletGoogleLayer
+                googleMapsLoaderConf={{
+                    KEY: GOOGLE_CLIENT_SIDE_API_KEY,
+                    REGION: countryCode,
+                    VERSION: 3.37,
+                }}
+                type="roadmap"
+            />
+            <Control position="topright">
                 <CopyToClipboard
                     text={window.location.href}
                     onCopy={() => toast('Copied search to clipboard')}
                 >
                     <Button
                         text="Share This Search"
                         onClick={noop}
-                        style={facilitiesMapStyles.copySearchButtonStyle}
+                        style={mapComponentStyles.copySearchButtonStyle}
                     />
                 </CopyToClipboard>
-                <ShowOnly when={this.state.disambiguationPopupIsOpen}>
+            </Control>
+            <ZoomControl position="bottomright" />
+            <MarkerClusterGroup
+                showCoverageOnHover={false}
+                removeOutsideVisibleBounds
+                spiderfyOnMaxZoom={false}
+                iconCreateFunction={(cluster) => {
+                    const clusterCount = cluster.getChildCount();
+                    const [iconClassName, iconSize] = (() => {
+                        if (clusterCount < 10) {
+                            return ['cluster-icon-one', [53, 53]];
+                        }
+
+                        if (clusterCount < 25) {
+                            return ['cluster-icon-two', [55, 55]];
+                        }
+
+                        if (clusterCount < 50) {
+                            return ['cluster-icon-three', [65, 65]];
+                        }
+
+                        if (clusterCount < 100) {
+                            return ['cluster-icon-four', [78, 78]];
+                        }
+
+                        return ['cluster-icon-five', [90, 90]];
+                    })();
+
+                    return L.divIcon({
+                        className: iconClassName,
+                        iconSize,
+                        html: `<span>${clusterCount}</span>`,
```

Comment:
Consider using the application font and tweaking the span style so that the numbers appear more centered.

```diff
diff --git a/src/app/src/components/FacilitiesMap.jsx b/src/app/src/components/FacilitiesMap.jsx
index 8669cc1..ca794f1 100644
--- a/src/app/src/components/FacilitiesMap.jsx
+++ b/src/app/src/components/FacilitiesMap.jsx
@@ -214,9 +214,9 @@ function FacilitiesMap({
                     })();
 
                     return L.divIcon({
-                        className: iconClassName,
+                        className: `cluster-icon ${iconClassName}`,
                         iconSize,
-                        html: `<span>${clusterCount}</span>`,
+                        html: `<span style="margin: -0.1rem; display: block;">${clusterCount}</span>`,
                     });
                 }}
                 onClusterClick={({
diff --git a/src/app/src/styles/css/leafletMap.css b/src/app/src/styles/css/leafletMap.css
index 71c6d98..aea2022 100644
--- a/src/app/src/styles/css/leafletMap.css
+++ b/src/app/src/styles/css/leafletMap.css
@@ -1,3 +1,7 @@
+.cluster-icon {
+    font-family: ff-tisa-sans-web-pro, sans-serif;
+}
+
 .cluster-icon-one {
     width: 53px;
     height: 53px;
```

#### Before
![Screen Shot 2019-07-30 at 16 13 52](https://user-images.githubusercontent.com/17363/62171799-0e60f180-b2e5-11e9-954b-da155a26eca4.png)

#### After

![Screen Shot 2019-07-30 at 16 13 30](https://user-images.githubusercontent.com/17363/62171807-1456d280-b2e5-11e9-8b04-5be91a3cdfa8.png)

Reply:
Cool -- I added this in b615822262e038e6fff06ba846b086e09a285f39

![Screen Shot 2019-08-01 at 12 52 18 PM](https://user-images.githubusercontent.com/4165523/62312116-5c980100-b45b-11e9-8181-12218d22fa03.png)